examples/prism-cmake.html

```diff
@@ -0,0 +1,86 @@
+<h2>Comments</h2>
+<pre><code># This is a comment</code></pre>
+
+<h2>Functions</h2>
+<pre><code>add_library(foo main.cpp)
+target_link_libraries(foo bar)
+</code></pre>
+
+<h2>Keywords</h2>
+<pre><code>CMAKE_COMPILER_IS_GNUG77
+CMAKE_COMPILE_PDB_OUTPUT_DIRECTORY
+CMAKE_COMPILE_PDB_OUTPUT_DIRECTORY_\w+
+CMAKE_CONFIGURATION_TYPES
+CMAKE_CPACK_COMMAND
+CMAKE_CROSSCOMPILING
+CMAKE_CROSSCOMPILING_EMULATOR
+CMAKE_CTEST_COMMAND
+CMAKE_CUDA_EXTENSIONS
+CMAKE_CUDA_HOST_COMPILER
+CMAKE_CUDA_SEPARABLE_COMPILATION
+CMAKE_CUDA_STANDARD
+CMAKE_CUDA_STANDARD_REQUIRED
+</code></pre>
+
+<h2>Properties</h2>
+<pre><code>CUDA_STANDARD
+CUDA_STANDARD_REQUIRED
+CXX_EXTENSIONS
+CXX_STANDARD</code></pre>
+
+<h2>Full example</h2>
+<pre><code>cmake_minimum_required(VERSION 3.13)
+
+project(crypto)
+
+add_library(base INTERFACE)
+target_compile_features(base INTERFACE
+    cxx_std_17
+    )
+
+add_subdirectory(helpers)
+add_subdirectory(msg)
+
+add_library(analyzers-obj OBJECT
+    CryptoAnalyzer.cpp
+    )
+
+target_include_directories(analyzers-obj
+    PUBLIC
+        ${CMAKE_CURRENT_SOURCE_DIR}
+    )
+
+find_package(predi REQUIRED)
+target_link_libraries(analyzers-obj
+    PUBLIC
+        base
+        predi::predi
+        crypto::helpers
+    )
+
+set_target_properties(analyzers-obj
+    PROPERTIES
+        POSITION_INDEPENDENT_CODE ON
+    )
+
+add_library(analyzers SHARED
+    $<TARGET_OBJECTS:analyzers-obj>
```

Comment:
The `<` should be escaped using `&lt;`.

Reply:
I guess the other one with `&gt;`? I still haven't figured out how to run this example, hence the trivial mistakes.

Reply:
> I still haven't figured out how to run this example

[As I said](https://github.com/PrismJS/prism/pull/1820#discussion_r266665156), you can't until we merge this PR.